src/test/java/com/metamx/common/logger/LoggerTest.java

```diff
@@ -26,4 +26,13 @@ public void testLogWithCrazyMessages()
     final Logger log = new Logger(LoggerTest.class);
     log.warn(message);
   }
+
+  @Test
+  public void testLegacyLogging()
+  {
+    final Logger log = new Logger(LoggerTest.class);
+    final Throwable throwable = new Throwable();
+    log.error("foo", throwable);
+    log.warn("foo", throwable);
+  }
```

Comment:
i think you are expecting IDE to flag these as deprecated. but since there are no asserts, its difficult for someone to understand what is being "tested" here. can you add comment telling why this test needs to exist or may be remove it since it can't really "fail" whether or not the thing you expected works or not.